HIP/hip-583.md

```diff
@@ -2,221 +2,268 @@
 
-As an illustration, a wallet or exchange could create and fund a new Hedera account based on the `public-address` alone. Once the transaction goes through the owner of the `public-address` user may use a wallet of their choice to submit a signed transaction (e.g. `CryptoTransfer`, `ContractCreate`, `ContractCall`, `EthereumTransaction` or `TokenAssociation`) to claim ownership of the account and carry out the desired transaction. The ownership claim and account completion will be seamless and transparent to the user.
+Since Hedera accounts require keys to ensure security, an `ECDSA` key matching the derived `evm address alias` must be
+provided in a future signed transaction issued by the owner of the account to take control of the account. Due to the
+nature of `ECDSA secp256k1` keys, the public key can be extracted from the signature and the signed bytes. It can then
+go through the derivation function to generate the associated `evm address`, which can be compared with the `alias` to
+prove the match and account ownership. This process will complete the creation of the account, taking it from a
+`Hollow Account` to a normal complete account with a valid public key.
 
-`CryptoTransfer` transactions should also support this case.
-This will support users, developers and exchanges who prefer to create and credit an accounts balance in one step.
+Note, in this case the final value of the account alias will remain as the `evm address alias` and not the `key alias`.
 
-`EthereumTransaction` transactions where value is transfered  (i.e `tx.to` and `tx.value` are set) should also support this case.
-This will support users of `Ethereum-native wallets` and eth dev tools who are accumstomed to creating and crediting an accounts balance in one step.
-In this case th `tx.to` value of the RLP encoded transaction would dictate the `public-address` to be used as an alias.
+`EthereumTransaction` transactions where value is transferred  (i.e `tx.to` and `tx.value` are set) should also support
+this case. This will support users of `Ethereum-native wallets` and eth dev tools who are accustomed to creating and
+crediting an account's balance in one step. In this case th `tx.to` value of the RLP encoded transaction would dictate
+the `evm address alias` to be used.
 
-### Allow CryptoCreate to automatically set alias
+### Setting `alias` with `CryptoUpdate`
 
-When `CryptoCreate.key` is an `ECDSA` key, the network node may calculate the `public-address` and automatically set the accounts alias value. 
+`HIP-32` supports updating the `alias` of an account after the account has been created. This HIP extends that
+support to include `evm address alias`es. As with `CryptoCreate`, ownership of the private key of the `ECDSA` public
+key associated with the `evm address` **must** be asserted on the `CryptoUpdate` by signing the transaction with
+that private key. The key used does **not** have to be in state on any account.
```

Comment:
We should highlight the restriction here for an account that already has a key alias (ED or EC) set.
Multiple alias in state should be outside of this HIP.

Reply:
Agree!